src/common/mmo.hpp

```diff
@@ -701,6 +701,20 @@ struct guild_castle {
 	int temp_guardians_max;
 };
 
+/// Guild Permissions
+enum e_guild_permission {
+	GUILD_PERM_INVITE	= 0x001,
+	GUILD_PERM_EXPEL	= 0x010,
+#if PACKETVER >= 20140205
+	GUILD_PERM_STORAGE	= 0x100,
```

Comment:
Shouldn't these be 0x001, 0x002, 0x004?

Reply:
Gravity... :-) This is how bitmasks work for one of their developers at least...

Reply:
Funny